python/wawenets/speech/T000053_Q159_D401_48.wav

```diff
@@ -0,0 +1 @@
+../../../speech/T000053_Q159_D401_48.wav
```

Comment:
I was afraid of this. `git` on windows doesn't always play nice with symlinks: https://stackoverflow.com/a/59761201/650427

the files are not that big. I think it is probably more user friendly to just copy the files instead of expecting people to deal with their git install.

I think the only reason the `.wav` files are part of the python package to begin with is for tests. I think what I will actually do is make the tests generate a couple files synthetically and then delete them.

how does that sound?

Reply:
I ended up removing one and keeping the other. 🤷